engine-dmn/feel-juel/src/main/java/org/operaton/bpm/dmn/feel/impl/juel/el/FeelFunctionMapper.java

```diff
@@ -27,24 +26,49 @@
 import org.operaton.bpm.dmn.feel.impl.juel.FeelEngineLogger;
 import org.operaton.bpm.dmn.feel.impl.juel.FeelLogger;
 
+/**
+ * A custom implementation of the `FunctionMapper` class for mapping FEEL (Friendly Enough Expression Language)
+ * functions to Java methods. This class provides support for resolving and executing FEEL functions.
+ */
 public class FeelFunctionMapper extends FunctionMapper {
 
+  // Logger instance for logging errors and messages
   public static final FeelEngineLogger LOG = FeelLogger.ENGINE_LOGGER;
 
-  protected static final SimpleDateFormat FEEL_DATE_AND_TIME_FORMAT = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss");
+  // Date and time format used for parsing FEEL date and time strings
+  protected static final String FEEL_DATE_AND_TIME_FORMAT = "yyyy-MM-dd'T'HH:mm:ss";
+
+  // Name of the JUEL method for date and time parsing
```

Comment:
As mentioned above I would like to have such descriptions as Javadoc comments instead of single line. Please change also for other docs added for fields.
Javadocs have nice IDE support. When we already add descriptions for fields, then they should be in Javadoc format.